aten/CMakeLists.txt

```diff
@@ -121,6 +121,10 @@ ENDIF()
 # Find the HIP package, set the HIP paths, load the HIP CMake.
 IF(WITH_ROCM)
   include(LoadHIP)
+  if (NOT PYTORCH_FOUND_HIP)
```

Comment:
Can replace PYTORCH_FOUND_HIP with HIP_FOUND.

Reply:
See my comments in cmake/public/LoadHIP.cmake